pages/dkp/kaptain/2.0.0/download/index.md

```diff
@@ -32,4 +32,4 @@ Kaptain install package placed in the local directory. See install instructions
 You will now see the `kubeflow-1.4.0_1.3.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

Comment:
This also needs to be changed for the new version, see below.    

Also, what comes in the AWS container image is the charts bundle, which isn't needed for a networked install, which is likely to be the most common type on AWS.

The only instructions that mention this file are the air-gapped install, which is a valid (but likely unusual) use case for Kaptain on AWS.   I am not sure what, or if anything should be changed here, just pointing out that a customer might be confused to download this 'kaptain-2.0.0.tgz' file and then not actually need to use it to install Kaptain.

```suggestion
You will now see the `kaptain-2.0.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

Reply:
@lukeogg can you please take a look at Craig's comment and let me know what I should change on an iterating PR?